src/core/main.c

```diff
@@ -1991,6 +2006,7 @@ int main(int argc, char *argv[]) {
                 arg_default_rlimit[j] = mfree(arg_default_rlimit[j]);
 
         arg_default_unit = mfree(arg_default_unit);
+        arg_confirm_spawn = mfree(arg_confirm_spawn);
```

Comment:
We'd crash here if arg_confirm_spawn was the non-allocated string `"/dev/console"`. See my comment above.